Content.Shared/_ES/Auditions/CharacterComponent.cs

```diff
@@ -0,0 +1,30 @@
+using Content.Shared.Humanoid;
+using Robust.Shared.Enums;
+using Robust.Shared.GameStates;
+
+namespace Content.Shared._ES.Auditions;
+
+/// <summary>
+/// This is used for marking the character of components.
+/// </summary>
+[RegisterComponent, NetworkedComponent]
+public sealed partial class CharacterComponent : Component
+{
+    [ViewVariables, DataField]
```

Comment:
you can remove all the ViewVariables attributes as they are just done automatically nowadays.